include/itkAttenuationImageFilter.hxx

```diff
@@ -277,13 +277,14 @@ AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::ThreadedGenerateD
             }
             float estimatedAttenuation = ComputeAttenuation(target, start);
 
-            // Compute mid-point index
-            target[m_Direction] = (start[m_Direction] + target[m_Direction]) / 2;
+            // Record this attenuation for both pixels of the pair
+            output->SetPixel(start, estimatedAttenuation);
             output->SetPixel(target, estimatedAttenuation);
```

Comment:
Does this result in the metric image pixel value at `target` being overwritten multiple times as pairs are computed?

Reply:
This is the pairs case, where only one pair per segment is done. Lines 228-268 are for per pixel estimation.